### PR TITLE
Christian - Add Update.java and UpdateCollection.java for Course Last Updated Feature

### LINKED ISSUE #17: LAST UPDATED FEATURE: Establish Class for the MongoDB Collection UpdateCollection.java
Establish the class for the MongoDB collection `UpdateCollection.java` for the `updates` collection based on the class `src/main/java/edu/ucsb/cs156/courses/collections/ConvertedSectionCollection.java` that forms the basis of the current `courses` collection. This is the class that will be used to do queries on the collection.

### PR DESCRIPTION
**Pull request by b4k3s**
In this PR, we add Update and UpdateCollection which are classes that are needed to implement the last updated feature which is a timestamp when a course was last updated and is stored in MongoDB. Also, the database collection `updates` for this feature was setup on MongoDB. Closes #17.

### CODE REVIEW

File: src/main/java/edu/ucsb/cs156/courses/jobs/UpdateCourseDataJob.java [hunk 1]
```diff
@@ -22,6 +23,26 @@ public class UpdateCourseDataJob implements JobContextConsumer {
...
+    //     } else {
+    //         uc.save(
+    //             Update.builder()
+    //                 .subjectArea(subjectArea)
+    //                 .quarterYYYYQ(quarterYYYYQ)
+    //                 .lastUpdate(ZonedDateTime.now())
+    //                 .build()
+    //         );
+    //     }
+    // }
```
Comment (pconrad) [inline src/main/java/edu/ucsb/cs156/courses/jobs/UpdateCourseDataJob.java]: We try not to put commented out code into PRs to main.

Please remove this.

**Review by pconrad — CHANGES_REQUESTED** [review]
Please remove commented out code

**Comment by b4k3s** [comment]
Was not planning on leaving it commented out. I just needed to make sure it was that function that was giving me issues because I could not test it locally.

File: src/main/java/edu/ucsb/cs156/courses/jobs/UpdateCourseDataJob.java (hunk 1 above)
Comment (b4k3s) [reply src/main/java/edu/ucsb/cs156/courses/jobs/UpdateCourseDataJob.java]: OK! I will remove this. I was not planning on this commit being merged. I cannot run the pitest command executed by the github action locally for some reason, and I just wanted to make sure that this was what was giving me errors.

**Review by pconrad — APPROVED** [review]
LGTM!

**Review by cameron-barrett — APPROVED** [review]
LGTM

**Pull request merged**